src/main/groovy/grails/plugins/mail/MailGrailsPlugin.groovy

```diff
@@ -47,16 +47,6 @@ sendMail {
 
     def license = "APACHE"
     def organization = [ name: "Grails Plugin Collective", url: "http://github.com/gpc" ]
```

Comment:
I wonder if removing the list of original developers violates the license on redistribution.
<img width="404" alt="image" src="https://github.com/spidasoftware/mail/assets/89933230/1849f7de-91ea-40a4-a0a5-b102cd4ada64">